src/devices/controlBoardCouplingHandler/ControlBoardCouplingHandler.h

```diff
@@ -0,0 +1,225 @@
+/*
+ * SPDX-FileCopyrightText: 2006-2021 Istituto Italiano di Tecnologia (IIT)
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef YARP_DEV_CONTROLBOARDREMAPPER_CONTROLBOARDREMAPPER_H
+#define YARP_DEV_CONTROLBOARDREMAPPER_CONTROLBOARDREMAPPER_H
+
+#include <yarp/os/Network.h>
+
+#include <yarp/dev/ControlBoardInterfaces.h>
+#include <yarp/dev/PolyDriver.h>
+#include <yarp/dev/IJointCoupling.h>
+#include <yarp/dev/ControlBoardInterfacesImpl.h>
+#include <yarp/dev/IPreciselyTimed.h>
+#include <yarp/os/Semaphore.h>
+#include <yarp/dev/IMultipleWrapper.h>
+
+#include <string>
+#include <vector>
+
+
+#ifdef MSVC
+    #pragma warning(disable:4355)
+#endif
+
+/**
+ * @ingroup dev_impl_remappers
+ * FIXME !!!!
+ * @brief `controlBoardCouplingHandler` : device that takes a list of axes from multiple controlboards and expose them as a single controlboard.
+ *
+ * | YARP device name |
+ * |:-----------------:|
+ * | `controlBoardCouplingHandler` |
+ *
+ *
+ *  Parameters required by this device are:
+ * | Parameter name | SubParameter   | Type    | Units          | Default Value | Required                    | Description                                                       | Notes |
+ * |:--------------:|:--------------:|:-------:|:--------------:|:-------------:|:--------------------------: |:-----------------------------------------------------------------:|:-----:|
+ * | axesNames     |      -         | vector of strings  | -      |   -           | Yes     | Ordered list of the axes that are part of the remapped device. |  |
+ *
+ * The axes are then mapped to the wrapped controlboard in the attachAll method, using the
+ * values returned by the getAxisName method of the controlboard. If different axes
+ * in two attached controlboard have the same name, the behaviour of this device is undefined.
+ *
+ * Configuration file using .ini format.
+ *
+ * \code{.unparsed}
+ *  device controlBoardCouplingHandler
+ *  axesNames (joint1 joint2 joint3)
+ *
+ * ...
+ * \endcode
+ *
+ * For compatibility with the controlBoard_nws_yarp, the
+ * networks keyword can also be used to select the desired joints.
+ * For more information on the syntax of the networks, see the
+ * controlBoard_nws_yarp class.
+ *
+ * \code{.unparsed}
+ *  networks (net_larm net_lhand)
+ *  joints 16
```

Comment:
Is this 16 correct?

Reply:
The documentation has been copied from `remoteControlBoardremapper` but it has to be completeley rewritten also because I am not sure which will be the parameters, please ignore it